spec/services/identity_linker_spec.rb

```diff
@@ -226,16 +226,17 @@
         end
       end
 
-      context 'the request is includes identity proofing and verified_at is not null' do
+      context 'the request includes identity proofing and verified_at is not null' do
         it 'does not set the timestamp' do
+          freeze_time
```

Comment:
Does this risk bleeding into tests run after this one? In other tests we use `around` like this, to ensure the teardown:

```rb
around do |example|
  feeze_time { example.run }
end
```

Reply:
Interesting. It definitely seems easier to just have the one line of code, though generally speaking I feel a little uneasy about any sort of side-effecty code like this inside a test case.

Reply:
I was hoping to have the `freeze_time` block around it, but the helper doesn't like nested time traveling blocks:

```
     RuntimeError:

       Calling `travel_to` with a block, when we have previously already made a call to `travel_to`, can lead to confusing time stubbing.

       Instead of:

          travel_to 2.days.from_now do
            # 2 days from today
            travel_to 3.days.from_now do
              # 5 days from today
            end
          end

       preferred way to achieve above is:

          travel 2.days do
            # 2 days from today
          end

          travel 5.days do
            # 5 days from today
          end
```